pkg/curatedpackages/bundle_test.go

```diff
@@ -0,0 +1,168 @@
+package curatedpackages
```

Comment:
could we test this from `curatedpackages_test`? We always try to do blackbox testing when possible